curriculum/challenges/english/01-responsive-web-design/css-piano/part-002.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 612e76cb2d27e318702a5117
+title: Part 2
+challengeType: 0
+dashedName: part-2
+---
+
+# --description--
+
+Add the text `Responsive Web Design Piano` within the title element.
```

Comment:
**Note**: One day, we should meet, and go through all the projects to uniform _ize_ (?) the titles.